hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/cluster/ITTestHoodieFlinkClustering.java

```diff
@@ -18,6 +18,11 @@
 
 package org.apache.hudi.sink.cluster;
 
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.api.TableResult;
+import org.apache.flink.table.api.ValidationException;
```

Comment:
Fix the import sequence.

Reply:
Done

Reply:
hudi package should be in the first block and isolated with other blocks.

Reply:
Done